package/feature/feature-nest-stock/src/stock.service.ts

```diff
@@ -265,6 +265,106 @@ export class StockService {
     return result;
   }
 
+  async drawStockInfo(stockId: string, body: Request.PostDrawStockInfo): Promise<Stock> {
+    const { userId } = body;
+    console.debug('drawStockInfo', { userId });
+    let result: Stock;
+
+    const session = await this.connection.startSession();
+    await session.withTransaction(async () => {
+      const stock = await this.stockRepository.findOneById(stockId, undefined, { session });
+      const user = await this.userRepository.findOne({ stockId, userId }, undefined, { session });
+
+      if (!stock.isTransaction) {
+        throw new HttpException('지금은 거래할 수 없습니다', HttpStatus.CONFLICT);
+      }
+
+      if (!user) {
+        throw new HttpException('유저 정보를 불러올 수 없습니다', HttpStatus.CONFLICT);
+      }
+
+      // 현재 라운드에 해당하는 시점의 idx
+      const timeIdx = Math.min(
+        Math.floor(getDateDistance(stock.startedTime, new Date()).minutes / stock.fluctuationsInterval),
+        9,
+      );
+
+      const nextTimeIdx = timeIdx + 1;
+
+      const DEFAULT_DRAW_COST = 1;
```

Comment:
```suggestion
      const DEFAULT_DRAW_COST = 300000;
```

정보 뽑기 비용이 1원으로 설정되어 있군요.
30만원으로 바꿀 수 있나요?

Reply:
이런 실수를.. 테스트한다고 낮춰놨는데 수정을 깜빡했군요 
감사해요!!👍👍